src/core/meshcop/dataset_manager_ftd.cpp

```diff
@@ -93,6 +93,12 @@ Error DatasetManager::ProcessSetOrReplaceRequest(MgmtCommand          aCommand,
         aInfo.mAffectsConnectivity = true;
     }
 
+    if ((dataset.Read<WakeupChannelTlv>(channelValue) == kErrorNone) &&
+        (channelValue.GetChannel() != Get<Mac::Mac>().GetWakeupChannel()))
+    {
+        aInfo.mAffectsConnectivity = true;
```

Comment:
Do we want to set this? 
This will cause an "ActiveDataset" SET to be converted to be delayed and applied as "PendingDataset" SET. Is this desirable in all situations?

Reply:
Right, this is to be decided in SPEC-1324. For now I'll remove it (it was not part of the prototype, but this file had important changes so I added now accidentally.